Updated copy framework template script

rohandudam opened the pull request:
Fixed and updated script to support recent architectural changes.

From root directory, run following command to execute the script correctly.
python utils/copy_framework_template.py -s . -d ../copy_pom_temp/

avinash010 reviewed (commented):
The changes look good. I tested it and the framework templates are getting copied fine.
Couple of issues 
1) Do we need os_details.config
2) I see some pylint issues with the file. Can you resolve those

rohandudam commented:
> The changes look good. I tested it and the framework templates are getting copied fine. Couple of issues
> 
> 1. Do we need os_details.config
> 2. I see some pylint issues with the file. Can you resolve those

- Checked we won't need os_details.config anymore. deleted file and removed it from copy template conf list.
- Fixed pylint issues related to deprecated optparser.

avinash010 commented:
Looks good @rohandudam
